spec/services/fuzzy_find/finder_service_spec.rb

```diff
@@ -111,7 +110,7 @@ def self.where(args)
                   needle: "Newish Person",
                   haystack_model: Person,
                   attribute: :name,
-                  addl_attribute: { groups: "1" }
+                  addl_attribute: { groups: ["1"] }
```

Comment:
So, I think changing this is causing tests to fail on Travis. What's going on here? Why did it change?